tests/unittests/tests-devfs/Makefile.include

```diff
@@ -1,2 +1,5 @@
 USEMODULE += vfs
 USEMODULE += devfs
+USEMODULE += random
+USEMODULE += auto_init
```

Comment:
AFAIK auto_init is disabled for the unit tests application

Reply:
removed